cot/src/form/fields.rs

```diff
@@ -1216,4 +1257,23 @@ mod tests {
         let value = f32::clean_value(&field);
         assert_eq!(value, Err(FormFieldValidationError::Required));
     }
+
+    #[test]
+    #[cfg(feature = "test")]
+    fn url_field_clean_value() {
```

Comment:
Can we add some more tests? In particular, ones that try to render the field? Similar to https://github.com/cot-rs/cot/blob/f9d7c43c639b9074efa9870bab1edfd96c50b29a/cot/src/form/fields.rs#L849-L867 .